dough-inventories/src/main/java/io/github/thebusybiscuit/dough/inventory/ChestMenu.java

```diff
@@ -585,4 +574,33 @@ protected void onClose(Player p) {
     public ChestMenu clone() {
         return new ChestMenu(this);
     }
+
+    public boolean isPlayerInventoryClickable() {
+        return this.playerInventoryClickable;
+    }
+
+    public Plugin getPlugin() {
+        return this.plugin;
+    }
+
+    public String getTitle() {
+        return this.title;
+    }
+
+    public Consumer<Player> getMenuOpeningHandler() {
+        return this.menuOpeningHandler;
+    }
+
+    public Consumer<Player> getMenuClosingHandler() {
+        return this.menuClosingHandler;
+    }
```

Comment:
All four should be nonnull, right?

Reply:
I suppose. I haven't touched the actual menus just yet